contracts/sources/obligation.move

```diff
@@ -330,7 +423,16 @@ module suilend::obligation {
         amount: u64,
     ) {
         let borrow_index = find_or_add_borrow(obligation, reserve, clock);
-        assert!(vector::length(&obligation.borrows) <= MAX_BORROWS, ETooManyBorrows);
+        let is_emode = is_emode(obligation);
+
+        if (is_emode) {
```

Comment:
i would do this check _after_ the borrow is processed. similar to the isolated check 
`if (isolated(config(reserve)) || obligation.borrowing_isolated_asset) {`. the logic is simpler that way. 

same comment for deposit

Reply:
done